client/app/bundles/momentCard/components/MomentCardMoods.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import css from './MomentCard.scss';
+
+type MomentCardMoodsState = {};
+
+type MomentCardMoodsProp = {
+  mood: any;
+};
+
+export default class MomentCardMoods
+  extends React.Component<MomentCardMoodsProp, MomentCardMoodsState> {
```

Comment:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.